packages/metrics/src/features/audit/handler.ts

```diff
@@ -50,7 +55,8 @@ export async function newAuditHandler(
         payload.additionalData
       )
     )
-    await writePoints(points)
+
+    await writePoints(points, transactionId)
```

Comment:
`writePoints` function is now in charge of never writing duplicates even if the handler is called multiple times (given transaction id stays the same)